src/components/PeopleTabel/PersonRow.tsx

```diff
@@ -0,0 +1,55 @@
+import classNames from 'classnames';
+import { Person } from '../../types/Person';
+import { PersonLink } from './PersonLink';
+import { useParams } from 'react-router-dom';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonRow: React.FC<Props> = ({ person }) => {
+  const { slug } = useParams();
+  const { mother, father } = person;
+  const selectedRow = slug === person.slug;
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({ 'has-background-warning': selectedRow })}
+    >
+      <td>
+        <PersonLink
+          name={person.name}
+          sex={person.sex}
+          slug={person.slug}
+        ></PersonLink>
```

Comment:
```suggestion
        <PersonLink
          name={person.name}
          sex={person.sex}
          slug={person.slug}
        />
```
You can self closed tag if you don't pass children